BackEnd/.gitignore

```diff
@@ -0,0 +1,31 @@
+HELP.md
+target/
+.mvn/
+!**/src/main/**
+!**/src/test/**
+
+### STS ###
+.apt_generated
+.classpath
+.factorypath
+.project
+.settings
+.springBeans
+.sts4-cache
+
+### IntelliJ IDEA ###
+.idea
+*.iws
+*.iml
+*.ipr
+
+### NetBeans ###
+/nbproject/private/
+/nbbuild/
+/dist/
+/nbdist/
+/.nb-gradle/
+build/
+
```

Comment:
Add .mvn/wrapper/maven-wrapper.jar into .gitignore

Reply:
I added '.mvn/' (third line) this should exclude everything under that path, included '.mvn/wrapper/maven-wrapper.jar'